gandi/cli/modules/paas.py

```diff
@@ -29,6 +32,67 @@ def type_list(cls, options=None):
         return cls.safe_call('paas.type.list', options)
 
     @classmethod
+    def attach(cls, name, vhost, remote_name):
+        """Attach an instance's vhost to a remote from the local repository."""
+        remote_url = cls.git_remote(name, vhost)
+
+        ret = cls.execute('git remote add %s %s' % (remote_name, remote_url,))
```

Comment:
I thought that it could raise an ugly TB in case the command failed but instead it displayed the git error message, which is fine indeed. So you can leave it as it is.